exercise/do-your-own-reduce.js

```diff
@@ -0,0 +1,44 @@
+/*
+## Do your own reduce
+
+### Instructions
+
+Create the function `Reduce` that receives a function and an array
+and works just like the method [].reduce.
+
+The use of `[].reduce` is forbidden for this exercise.
+
+### Notions
+
+- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce
+
+*/
+///*/// ⚡
+
+///*/// ⚡
+export const tests = []
+const t = (f) => tests.push(f)
+
+let arr1 = [1, 2, 3, 4]
+let arr2 = [5, 10, 31, 7]
+const randArr = [...new Array(10)].map(() => Math.round(Math.random() * 15))
+
+const adder = (a, b) => a + b
+const multiplier = (a, b) => a * b
```

Comment:
I think we should have a test that is not a simple number opperation.
Add one that use strings concatenation and maybe objects:
```js
const joiner = (a, b) => `${a}, ${b}` // this is a good usecase of reduce over fold
const merger = (a, b) => ({...a, ...b})
```